core/runtime/ops.rs

```diff
@@ -872,4 +881,35 @@ mod tests {
     )?;
     Ok(())
   }
+
+  #[op2(core, fast)]
+  pub fn op_state_rc(state: Rc<RefCell<OpState>>, value: u32) -> u32 {
```

Comment:
Can we prevent this somehow? `Rc<RefCell<OpState>>` shouldn't be used in a sync op (`&mut OpState` should be used instead`)

Reply:
Definitely -- in this case I wanted to make sure it was working. I think it may be best to add a post-processing step when I add async support that denies this and clean it up, and then migrate this test to async ops. I can make sure that I add a TODO in the roadmap to clean this up later.

Reply:
Sounds good to me